content/docs/command-reference/push.md

```diff
@@ -66,10 +65,10 @@ The `dvc status -c` command can list files tracked by DVC that are new in the
 cache (compared to the default remote.) It can be used to see what files
 `dvc push` would upload.
 
-If one or more `targets` are specified, DVC only considers the files associated
-with them. Using the `--with-deps` option, DVC tracks dependencies backward from
-the target [stage files](/doc/command-reference/run), through the corresponding
-[pipelines](/doc/command-reference/pipeline), to find data files to push.
+If one or more `targets` are specified, DVC only considers the corresponding
+files. Note that DVC supports granularity as well: the targets may be files or
+directories found inside a directory that is
```

Comment:
`files` but then `files or directories`, seems confusing :slightly_frowning_face:

Reply:
Sorry, I don't understand the confusion, can you elaborate? These are two separate sentences:

> If one or more `targets` are specified, DVC only considers the corresponding **files**.
> Note that DVC supports granularity as well: the targets may be **files or directories** found inside a directory that is [tracked as a whole](/doc/command-reference/add#example-directory).

This granularity is a little confusing conceptually, maybe there's no way around that (unless we decide not to even mention it, of course, but then https://github.com/iterative/dvc.org/pull/1384#discussion_r453969894).